atomic_reactor/inner.py

```diff
@@ -56,7 +56,11 @@ class BuildResults(object):
 
 
 class BuildResultsEncoder(json.JSONEncoder):
-    def default(self, obj):
+    def __init__(self, **kwargs):
+        kwargs['default'] = self._default
+        super(BuildResultsEncoder, self).__init__(**kwargs)
```

Comment:
I'd probably just disable pylint check here, JSONEncoder states clearly that `default` should be implemented in subclass.

Doesn't this fix the warning?
```diff
- def _default(self, obj):
+ def _default(self, o):
```

Reply:
Ok (changing the signature did not fix it)